src/Tribe/Admin/Manager/Service_Provider.php

```diff
@@ -138,7 +138,7 @@ public function add_attendees_view_button() {
 		// If this is not the Admin Edit page or not the singular frontend view, bail out.
 		if (
 			! ( is_admin() && 'edit' == tribe_get_request_var( 'action' ) )
-			&& ! is_single()
+			&& ! is_singular()
```

Comment:
Should we get the array of CPTs supporting tickets and use as parameter?

Reply:
That could be a good idea to prevent the tickets queries running.

Reply:
Added